src/components/Header/Header.stories.mdx

```diff
@@ -47,13 +48,44 @@ static component with no functionality.
 
 ## Accessibility
 
-TBD
+The `Header` uses `react-focus-lock` to trap (or loop) focus within its various dropdowns:
+the login submenu, the search form, and the mobile navigation submenu. This is to
+make keyboard navigation easier for users. Trapping focus means that when a parent
+element is clicked and its dropdown becomes visible, focus remains among its children
+while the user tabs. The user may close the dropdown and escape the focus trap by clicking
+on the parent element once again, or by using the escape key.
+
+This is how the original NYPL `Header` was implemented and its Reservoir counterpart
```

Comment:
I don’t feel this comparison between the new and the old is necessary.  When the current implementation header is gone, this language will no longer be relevant.  Perhaps change the language to say that the current tabbing pattern is what NYPL implemented when the header was built in 20xx and then go on to talk about the _currently recommended_ patterns and plans for the future.

@EdwinGuzman Thoughts?

Reply:
I agree and something I didn't consider because we are still in the "old vs new" `Header` implementations. If we are in this state of comparison then this description _might_ be helpful and will also serve as a reminder that we need to revisit the focus trap implementation.

Otherwise, we can go with your suggestion and get the message out that we should forget about the older one and update everything moving forward.

Reply:
Let’s leave this alone for now.  Yes, the pattern does need to be changed, but we can save that change (both to the functionality and the docs) for the header overhaul that will happen in the distant future.